NaviGator/utils/navigator_gui/navigator_gui/host_monitor.py

```diff
@@ -71,5 +73,5 @@ def publish(self, _: rospy.timer.TimerEvent) -> None:
 
 if __name__ == "__main__":
     monitor = HostMonitor()
-    rospy.Timer(rospy.Duration(10), monitor.publish, oneshot=False)
-    rospy.spin()
+    rclpy.Timer(rclpy.Duration(10), monitor.publish, oneshot=False)
```

Comment:
Timers aren't created in the same way either, you want to use your node to make timers